scripts/pyro_camera_server.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+"""
+Script to run a Pyro camera server. This should be run on the control computer for a distributed
```

Comment:
This runs on the main computer, or on one of the camera computers? The name makes it sound like it is on each camera, but the comment makes it sound like the main computer.

Reply:
The camera computers. That's what I meant by 'control computer for a distributed camera'.

Reply:
Reworded slightly for avoidance of doubt.